src/projects/EnsureThat/Enforcers/EnumerableArg.cs

```diff
@@ -52,15 +52,9 @@ public IEnumerable<T> SizeIs<T>([ValidatedNotNull, InstantHandle]IEnumerable<T>
         {
             Ensure.Any.IsNotNull(value, paramName);
 
-#if NETSTANDARD1_1
             var count = value.LongCount();
 
             if (count != expected)
-#else
-            var count = value.LongCount();
-
-            if (count != expected)
-#endif
```

Comment:
Let's decouple this change, it looks like the intent was to use Count

Reply:
Done